src/app/backend.service.ts

```diff
@@ -0,0 +1,19 @@
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class BackendService {
+
+  url = /* 'http://localhost:9090'; */ '/api';
```

Comment:
It would be great if you could replace this with an environment variable.

Otherwise, this PR is good.

Reply:
Moved the variable to environments. Use --prod to use '/api' path instead of localhost